include/wx/msw/ole/activex.h

```diff
@@ -23,6 +23,7 @@
 
 #include "wx/msw/ole/oleutils.h" // wxBasicString &c
 #include "wx/msw/ole/uuid.h"
+#include "wx/msw/private/comptr.h"
```

Comment:
We can't include private header from a public one, so we need to move `wxCOMPtr` to a public header first.